init/initialize.sh

```diff
@@ -97,6 +97,15 @@ sed -e "s;RSYNCDIR_SS;$RSYNCDIR_SS;" \
     -e "s;RSYNCDIR_NPAD;$RSYNCDIR_NPAD;" \
     -e "s;RSYNCDIR_PTR;$RSYNCDIR_PTR;" \
     $SLICEHOME/conf/rsyncd.conf.in > /etc/rsyncd.conf
+
+# Allow CIRA rsync access to the paris-traceroute module at their own sites
+for site in "yyz01 yyc01 yul01"; do
```

Comment:
When I run a simple for loop from the command line with the sites in quotes it treats the entire string as a single element. If we remove the quotes, I think it iterates over each site which looks like what we want. Was there a reason for the quotes?

    for site in "yyz01 yyc01 yul01" ; do echo $site ; done
    yyz01 yyc01 yul01

    $ for site in yyz01 yyc01 yul01 ; do echo $site ; done
    yyz01
    yyc01
    yul01